template/web/src/store/createWagmiConfig.ts

```diff
@@ -13,7 +21,14 @@ export function createWagmiConfig(projectId: string) {
       },
       {
         groupName: 'Other Wallets',
-        wallets: [rainbowWallet, metaMaskWallet],
+        wallets: [
+          rainbowWallet,
+          metaMaskWallet,
+          trustWallet,
+          braveWallet,
+          ledgerWallet,
+          walletConnectWallet,
```

Comment:
Wallet Connect adds trash in the HTML, let's not add it. We can change our mind after they clean up their library.